### PR TITLE
Refactor, Add tests, Add test coverage, Add badge to readme

### PR DESCRIPTION
**Pull request by ashwinbhaskar**
The failure threshold is set to 85% 
Addresses issue - https://github.com/siftrics/sight-clojure/issues/2

### CODE REVIEW

**Comment by ashwinbhaskar** [comment]
@siftrics Have added tests. Covers 85.75% of the code. Also did some refactoring. Can you once check if it works and update it here?

**Comment by siftrics** [comment]
@ashwinbhaskar Gonna get to this today!

**Comment by ashwinbhaskar** [comment]
@siftrics the response now looks like this (using clojure keywords - which is the idiomatic way)

```
{
      :pages : [
        {
          :error: "",
          :file-index: 0,
          :page-number: 1,
          :number-of-pages-in-file: 3,
          :recognized-text: [
                                          {:top-left-y 301,
                             :bottom-right-y 340,
                             :bottom-left-x 1318,
                             :top-right-x 1616,
                             :bottom-left-y 340,
                             :top-right-y 301,
                             :top-left-x 1318,
                             :bottom-right-x 1616,
                             :confidence 0.11501831809890184,
                             :text "foos"}]
        },
        
      ]
}
```

So, you might have to bump up the version of the  client.

File: project.clj
Comment (siftrics) [inline project.clj:2]: Let's bump the version to 1.1.0.

**Review by siftrics — CHANGES_REQUESTED** [review]
@ashwinbhaskar Finally got a chance to review this. All the code changes look great.

Before merging, in addition to bumping the version to 1.1.0 in project.clj, can you please make these changes:

- **README.md**
  - Lines 12, 18, 24, and 33: Bump version to 1.1.0.
  - Lines 51 and 98: Make sure the usage example reflects the changes from this PR. You can update the example, if necessary.
  - Lines 56-78 and 76-90: Either change these examples to use the new keywords (`:recognized-text`, etc.), or write a short sentence stating how to access the fields now.
  - Line 111: You can mention you are the lead maintainer of this repository, if you want.

(I couldn't make inline comments because README.md wasn't modified near these lines)

File: project.clj
Comment (ashwinbhaskar) [reply project.clj:2]: done

**Comment by ashwinbhaskar** [comment]
@siftrics updated version, usage and maintainer info in readme

**Review by siftrics — APPROVED** [review]

**Pull request merged**
